ddp-workspace/projects/ddp-prion/src/assets/i18n/en.json

```diff
@@ -445,6 +446,39 @@
         "E.PO_BOX.INTERNATIONAL": "Cannot verify international PO Box. Please note, this can show up as an error on a successful request."
       }
     },
-    "TooltipAlt": "Information"
+    "TooltipAlt": "Information",
+    "CookiesBanner": {
+      "Message_First": "We use cookies to evaluate site usage, improve performance, and ensure proper function of the website.",
+      "Message_Second" : "Please review our",
+      "Message_Third": "to learn more about the cookies we use.",
+      "Policy": "Privacy Policy",
+      "Preferences": "Manage Preferences",
+      "Reject": "Reject Cookies",
+      "Accept": "Accept Cookies"
+    },
+    "CookiesModal": {
+      "Title": "Manage Cookie Preferences",
+      "Privacy": "Your Privacy",
+      "Privacy_Text": "The Prion Registry collects cookies for functional and analytical purposes. You can find out more and change our default cookie settings by clicking on the tabs to the left.Please note that functional cookies must be placed to allow proper function of the website.",
```

Comment:
Missing a space before "Please note that".  Also, the tabs are only on the left at certain screen sizes, so it would be good to make the text reflect that somehow.  Maybe just leave it as "by clicking the tabs" and don't specify where the tabs are.

Reply:
Done in baa19cc

Fixed text